Dockerfile.prod

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:20.04
+
+# Install dependencies and tini.
+RUN apt-get update \
+    && apt-get --no-install-recommends -y install libssl-dev libssl1.1 \
+    && rm -rf /var/lib/apt/lists/*
+
+# Run as unprivileged user.
+RUN groupadd --system --gid 1001 nimiq \
+    && adduser --system --home /home/nimiq --uid 1001 --gid 1001 nimiq
+USER nimiq
+
+# Change homedir to nimiq
+WORKDIR /home/nimiq
+
+# Create nimiq directory for configuration
+RUN mkdir -p /home/nimiq/.nimiq
+
+# Set default config can be overwritten by mounting
+COPY ./client.toml /home/nimiq/.nimiq
```

Comment:
not sure where this client.toml is

Reply:
It's in the root of the repo: https://github.com/nimiq/core-rs-albatross/blob/albatross/client.toml